quarto-render/README.md

```diff
@@ -0,0 +1,103 @@
+# A Github Action to Render Quarto Documents
+
+This repository stores the `quarto-render` [GitHub Action](https://github.com/features/actions) for use with [Quarto](https://quarto.org/) projects. This action will call `quarto render` on the root directory of the repo. It also has example workflows for some common uses centered around web publishing.
+
+## Examples
+
+### Rendering a static `quarto` site and pushing to `gh-pages` branch
+
+If your quarto project has `freeze: true`, you can use this Github action to render your documents and publish your workflow to Github pages by using something like the following workflow:
+
+```yaml
+# .github/workflows/render-and-publish.yaml
+name: Render and deploy Quarto files
+on:
+  push:
+  pull_request:
+
+jobs:
+  quarto-render-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Install Quarto and render"
+        uses: pommevilla/quarto-render@main
+
+      - name: "Deploy to gh-pages"
+        uses: peaceiris/actions-gh-pages@v3
+        if: github.ref == 'refs/heads/main'
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./_site
+```
+
+- See [here](https://github.com/pommevilla/friendly-dollop) for an example page that runs this workflow with `freeze: true` that has both Python and R code.
+- Read more about using freeze in your Quarto project [here](https://quarto.org/docs/books/book-authoring.html?q=freeze#freezing).
+
+### Installing Python or R dependencies before using `quarto-render` and publishing results to the `gh-pages` branch.
```

Comment:
We could probably create some example workflow to detail this. 
Like in https://github.com/r-lib/actions/tree/master/examples

I bet it could be more findable than the action folder README. 

What do you think ? 

We could also probably have this repo automatically tests those example workflows 🤔

Reply:
I actually had some tests set up in the [old `quarto-render` repo](https://github.com/pommevilla/quarto-render/actions). All they did was test that the action was working for sites. There was a test for frozen sites and for a site loading an `renv` file. I'll port them over here.

When I eventually get the render output argument going, I'll test for that stuff, too. What other test cases did you have in mind?

Reply:
And yes, I think that's a good idea! I was dumping everything in the README because I was copying [this repo](https://github.com/peaceiris/actions-gh-pages), but I think you're idea is better. I'll move stuff over.

Reply:
> What other test cases did you have in mind?

Nothing specific just simple test, or examples to be sure our action is still working.